forte/data/readers/classification_reader.py

```diff
@@ -0,0 +1,245 @@
+            if self.configs.one_based_index_label:
+                # adjust label encoding if dataset digit label is one-based
+                class_id = int(df_dict["label"]) - 1
+            else:
+                class_id = int(df_dict["label"])
+        else:
+            # decode string label from dataset into digit label
+            class_id = self._class2index[df_dict["label"]]
+        # initialize all ontologies instances as data pack module
+        # to add corresponding variables and functions
+        for (
+            ontology_field,
+            (start_idx, end_idx),
+        ) in input_ontology_indices.items():
+            path_str, module_str = ontology_field.rsplit(".", 1)
+            mod = importlib.import_module(path_str)  # sentence ontology module
+            entry_class = getattr(mod, module_str)
+            entry_class(pack, start_idx, end_idx)
+        # for now, we use document to store concatenated text and set the class here
+        doc = Document(pack, 0, input_ontology_indices[input_ontologies[-1]][1])
+        doc.document_class = [self.configs.index2class[class_id]]
+
+        pack.pack_name = line_id
+        yield pack
+
+    @classmethod
+    def default_configs(cls):
+        r"""This defines a basic configuration structure for classification dataset reader.
+
+        Here:
+            - forte_data_fields: these fields provides one-to-one
+            correspondence between given original dataset data fields and
+            labels/ontologies. For data fields without usage, user can
+            specify None for them.
+            - index2class: a dictionary that maps from zero-based indices to
+                classes
+            - input_ontologies: a list of ordered input ontologies that
+                user want to concatenate into an input text.
+            - digit_label:  boolean value that specifies whether label in dataset is digit.
+            - one_based_index_label: boolean value that specifies if dataset
+                provides one-based digit label.
+                True for one-based index, false otherwise.
+            - skip_k_starting_lines: many datasets' first line are columns
+                names, set it to 1 if it's the case. Otherwise set to 0.
+                User can also set it to other positive integers to skip
+                multiple lines.
+        """
+        return {
+            "forte_data_fields": [
+                "label",
+                "ft.onto.base_ontology.Title",
+                "ft.onto.base_ontology.Document",
+            ],
+            "index2class": None,
+            "input_ontologies": [
+                "ft.onto.base_ontology.Title",
+                "ft.onto.base_ontology.Document",
+            ],
+            "digit_label": True,
+            "one_based_index_label": True,
+            "skip_k_starting_lines": 1,
+        }
+
+
+def generate_text_n_input_ontology_indices(
+    input_ontologies, forte_data_fields_dict
+):
```

Comment:
return type of arguments

Reply:
added return type.

Reply:
need return at the function signature too

Reply:
where's the return type

Reply:
Please resolve this

Reply:
Return type is needed for the function annotation, don't only the docstring.

def generate_text_n_input_ontology_indices(
    text_fields: List[str], forte_data_fields_dict: Dict[str, str]
) --> <return type here>: